code/game/objects/random/random.dm

```diff
@@ -928,13 +928,13 @@
 	)
 
 /obj/random/highvalue
-	name = "random high valuable item"
-	desc = "This is a random high valuable item."
+	name = "random high value item"
+	desc = "This is a random high value item."
 	icon = 'icons/obj/coins.dmi'
 	icon_state = "coin_diamond_heads"
 	problist = list(
 		/obj/item/bluespace_crystal = 5,
-		/obj/item/stack/telecrystal{amount = 10} = 5,
+		/obj/item/stack/telecrystal{amount = 15} = 0.5,
```

Comment:
That's a nogo with the current vault security level.
Antags rushing the vault is already a meme, which won't get better by increasing the tc amount in a stack by 50%